float32 x
float32 y
float32 yaw
uint8 deliv_num
